root/vite.config.ts

```diff
@@ -98,7 +98,7 @@ export default defineConfig(({ mode }) => {
           runtimeCaching: [
             {
               urlPattern: ({ event }) => event.request.mode === "navigate",
-              handler: 'NetworkFirst',
+              handler: "NetworkFirst",
```

Comment:
@nichnarmada Do we check single/double quotes in linting rules?

Reply:
Yep I added that to make it all consistent in eslint